extensions/libraries/redcore/layouts/webservice/documentation.php

```diff
@@ -20,6 +20,7 @@
 $translationFallback = RBootstrap::getConfig('enable_translation_fallback_webservices', '1') == '1' ? JText::_('JENABLED') : JText::_('JDISABLED');
 $defaultLanguage = RTranslationHelper::getSiteLanguage();
 $defaultFormat = RBootstrap::getConfig('webservices_default_format', 'json');
+$defaultStatefulness = RBootstrap::getConfig('webservices_stateful', '1') == '1' ? JText::_('JENABLED') : JText::_('JDISABLED');
```

Comment:
just one more thing...
please change RBootstrap::getConfig('webservices_stateful', '1') to RBootstrap::getConfig('webservices_stateful', '0') since on all other cases and by default you have put it as 0 as default behavior...
or better yet, change all other and default value to 1, since that is the default behavior we have now, or else all other sites and projects will have to adapt to this change.